test/unit/loki.test.ts

```diff
@@ -11,6 +11,7 @@ import mockContext from "../fixtures/context/fetchContext1.json"
 
 describe("Loki", () => {
   // Taken from context.test.ts snapshots
+  // @ts-expect-error
```

Comment:
I am assuming this is the same issue I ran into when making my objects like you did with zod and it makes typescript unhappy. I don't think we should ignore errors like this. We should setup our objects to be compliant or it could get broken by future typescript updates and cause us headaches later on.

Reply:
hmm yeah, instead of using a JSON file, we should be outputting a `.ts` file with `as const` so that the type gets narrowed correctly, did you want to take on that refactor? It'll involve modifying `context.test.ts` to output the changed fixtures.